src/components/CurrencyInput/CurrencyInput.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import { withTheme } from 'emotion-theming';
+
+import IconInput from '../IconInput';
+import { themePropType } from '../../util/shared-prop-types';
+
+const currencyBaseStyles = ({ theme }) => css`
+  color: ${theme.colors.b700};
+  line-height: 16px;
+`;
+
+const CurrencyIcon = styled('span')`
+  ${currencyBaseStyles};
+`;
+
+/**
+ * SearchInput component for forms.
+ */
+const CurrencyInput = ({ currencyPosition, currency, theme, ...props }) => (
+  <IconInput
+    iconLeft={({ className }) =>
```

Comment:
#121 combines the `<IconInput>` and `<Input>` components. So you should update this code to: 

```js
<Input
  prefix={({ className }) =>
    currencyPosition === 'left' && (
      <CurrencyIcon className={className}>{currency}</CurrencyIcon>
    )
  }
  suffix={({ className }) =>
    currencyPosition === 'right' && (
      <CurrencyIcon className={className}>{currency}</CurrencyIcon>
    )
  }
  {...props}
>
  {children}
</Input>
```